src/pages/planters/[planterid].js

```diff
@@ -273,6 +275,18 @@ export async function getServerSideProps({ params }) {
       associated_organizations,
     );
     props.planter.species = await utils.requestAPI(species);
+    if (props.planter.featuredTrees?.trees?.length) {
```

Comment:
I expected to see a lat/log in the planter data returned from `planter = await utils.requestAPI(url)` above, but did not.  I found that pulling lat/lon from their featuredTreed could be a solution to this.  The down-side would be if they plant in multiple counties it would just pull from the first.  Using the lat/lon from their profile picture could be the best solution.  

Please suggest another geo to use if appropriate.  Thanks!

Reply:
@stellarjaysoftware you read my mind :) yes, this is the way I was thinking, let's do it, as a temporary solution.

To set the planter country information, we need the admin panel team to do it on the admin dashboard, but that takes time, so let's go first.